model/k8s/registry.yaml

```diff
@@ -153,6 +177,18 @@ groups:
         brief: >
           The name of the DaemonSet.
         examples: ['opentelemetry']
+      - id: k8s.horizontalpodautoscaler.uid
```

Comment:
see my other comment, I think `k8s.hpa` is fine (assuming it's a common abbreviation in k8s world)

Reply:
yeah, it's a short name that can be used with kubectl

Reply:
Alright, moved it back to `hpa`, thank's!